src/components/organisms/AdminVenues/AdminVenues.scss

```diff
@@ -15,6 +16,24 @@
     font-weight: bolder;
   }
 
+  &__button {
+    height: 45px;
+    background-color: $primary;
+    padding: 1rem;
```

Comment:
There is `%` `px` and `rem` in a single file for different padding values. Very confusing.